### PR TITLE
Release 0.28

### PR DESCRIPTION
**Pull request by bote795**
## Description

update main with versions from 0.28 release

## Related Issues

https://github.com/astronomer/issues/issues?q=is%3Aissue+label%3A0.28.0+

## Testing

0.28 is being tested by qa

### CODE REVIEW

File: configs/local-dev.yaml [hunk 1]
```diff
@@ -68,8 +68,11 @@ astronomer:
       requests:
         cpu: "0m"
         memory: "0Mi"
-
     config:
+      registry:
+        protectedCustomRegistryEnabled: true
+      deployments:
+        enableUpdateDeploymentImageEndpoint: true
       emailConfirmation: false
```
Comment (vishwas-astro) [inline configs/local-dev.yaml:76]: Do we need the above 4 lines? Is this a dev only set up which should get reflected in our dev cluster?

**Comment by danielhoherd** [comment]
This merge is going the wrong way. We should not be merging the latest release into main, we should be merging main into the latest release branch. It's not important that all those rc changes go into the main branch history, and the other things that were bumped should have been bumped in main to begin with.

**Comment by vishwas-astro** [comment]
@danielhoherd is correct. We should update main with the Houston and Astro UI upgrades first and then pick them to release branch

**Comment by vishwas-astro** [comment]
We should still update main. No opinion on whether you do it this way or raise a different PR against main.

File: configs/local-dev.yaml (hunk 1 above)
Comment (bote795) [reply configs/local-dev.yaml:76]: not sure why this was added.. but ill just close the issue and just update the versions..

**Comment by bote795** [comment]
created following?: https://github.com/astronomer/astronomer/pull/1367

**Pull request closed without merging**